gwakeonlan/import_ethers.py

```diff
@@ -0,0 +1,40 @@
+# @license AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
```

Comment:
If you would change the copyright part to GPL-2+ like the rest of the code it would be better, without any need to relicense the application.